erts/emulator/test/busy_port_SUITE.erl

```diff
@@ -255,7 +255,7 @@ no_trap_exit(Config) when is_list(Config) ->
     Pid = fun_spawn(fun no_trap_exit_process/3, [self(), linked, Config]),
     receive
         {Pid, port_created, Port} ->
-            io:format("Process ~w created port ~w", [Pid, Port]),
+            ct:log("Process ~w created port ~w", [Pid, Port]),
```

Comment:
Any reason why you made this change? The logs end up in the same place anyway.

Reply:
I thought that test cases are expected to use `ct:log` (or, `print`, `pal`) to make the intent explicit. 
In my experience, I've been using `erlang:display` for hard debugging (when I want to avoid any log redirection), then `io:format(user, ...` for slightly softer debugging (and some convenience for argument format), but I could not see any value in having `io:format` in test suites. To my understanding, it should be one of `ct:pal/log/print`.

Reply:
I agree with your reasoning and I do the same as you. I was only curious if you had any other reason.